coretex/cli/modules/project_utils.py

```diff
@@ -23,9 +36,31 @@ def selectProjectType() -> ProjectType:
     return selectedProjectType
 
 
+def selectProjectVisibility() -> ProjectVisibility:
+    availableProjectVisibilities = {
+        "Private": ProjectVisibility.private,
+        "Public": ProjectVisibility.public,
+    }
+
+    choices = list(availableProjectVisibilities.keys())
+    selectedChoice = ui.arrowPrompt(choices, "Specify visibility of project:")
+
+    selectedProjectVisibility = availableProjectVisibilities[selectedChoice]
+    ui.stdEcho(f"You've chosen: {selectedChoice}")
+    return selectedProjectVisibility
+
+
+def changeProjectVisibility(id: int, visibility: ProjectVisibility) -> NetworkResponse:
+    return networkManager.post("entity-visibility", {
+        "entity_id": id,
+        "type": EntityVisibilityType.project,  # this number represents Project Entity enum value on backend side
+        "visibility": visibility,
+    })
+
+
 def promptProjectCreate(message: str, name: str) -> Optional[Project]:
     if not click.confirm(message, default = True):
-        raise RuntimeError
+        return None
```

Comment:
What does this affect?

Reply:
Well in initial implementation for some weird reason I've implemented that If user when prompted to create a new project (if project with specified name is not found) selects No as an answer it will raise RuntimeError, but since there was no description and it was only RuntimeError it would act like return but in the logs you can clearly see the error was raised here, so I've modified this to work as expected. (If user selects no when asked to create new project command will simply stop without any errors in logs and etc.)